src/main/groovy/info/solidsoft/gradle/pitest/PitestPlugin.groovy

```diff
@@ -175,8 +175,11 @@ class PitestPlugin implements Plugin<Project> {
         task.additionalClasspath.setFrom({
             List<FileCollection> testRuntimeClasspath = (extension.testSourceSets.get() as Set<SourceSet>)*.runtimeClasspath
             FileCollection combinedTaskClasspath = project.objects.fileCollection().from(testRuntimeClasspath)
+            List<String> listOfStrings = extension.
+                    fileExtensionsToFilter.
+                    getOrElse([])
```

Comment:
Good catch, it doesn't need to be calculated multiple times. Nevertheless `listOfStrings` seems to be a default name generated by IDE. `fileExtensionsToFiler`?

Reply:
Yes, very much an oversight on my side.